python/psautohint/otfFont.py

```diff
@@ -499,33 +549,45 @@ def convertBezToT2(bezString):
             lastPathOp = token
         elif token == "rb":
             lastPathOp = token
-            try:
-                i = hhints.index(argList)
-            except ValueError:
-                i = len(hhints)
+            if in_mm_hints:
```

Comment:
The same logic is repeated four times. Maybe better to factor it out as a function?